edn_format/char.py

```diff
@@ -1,6 +1,12 @@
 # -*- coding: utf-8 -*-
 
-class Char(str):
+import sys
+
+if sys.version_info[0] == 3:
```

Comment:
Can we refactor this into a common location? Since we're doing this in tests also.

Reply:
Done. It’s indeed cleaner.